presto-client/src/main/java/com/facebook/presto/client/StatementClientV1.java

```diff
@@ -91,6 +92,9 @@
             "/" +
             firstNonNull(StatementClientV1.class.getPackage().getImplementationVersion(), "unknown");
 
+    //add new elements as a comma-separated list, eg. "path, foo, bar"
+    private static final String CLIENT_CAPABILITIES = "path";
```

Comment:
why not to keep it as a plain list (or even set) instead of string?

Reply:
I think it's simplest to use a string when the header is going to be set to a string anyway.

Reply:
I also think that we shouldn't maintain the capabilities as a string. How about we have an enum, say `ClientCapabilities`, you can then create this static final String with `Joiner.on(",").join(ClientCapabilities.values())`. You can use that enum in other parts of the code (e.g., you hardcode "path" in `SetPathTask`).